teachopencadd/talktorials/T038_protein_ligand_interaction_prediction/talktorial.ipynb

```diff
@@ -112,7 +112,7 @@
     "\n",
```

Comment:
<p>No linking of talktorials to avoid potential issues here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>